INSTALL.md

```diff
@@ -334,6 +334,7 @@ Creating the database
         You can import this sample dump in the same way as the full dump above.
 
     4.  Build materialized tables (optional but recommended)
+        <a name="build-materialized-tables"></a>
```

Comment:
Is it doable to just anchor to a span around the actual header?

Reply:
Does a span show differently? If so, ok :)

Reply:
No, I mean the source Markdown file would be less readable.

Reply:
Not sure I agree, but it's also not too important so let's let it be as is :)